src/components/map/PolygonMarkerMap.tsx

```
@@ -85,6 +88,17 @@ const PolygonMarkerMap = (props: Props) => {
     }
   }, [currentUserLocation])
 
+  useEffect(() => {
+    checkForExistingCoords()
+  }, [])
+
+  const checkForExistingCoords = async () => {
+    if (previousData && previousData.location && previousData.location.coordinates) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)